fix chicken version for chicken 5 style import

farseerfc opened the pull request:
This try to install depends into a ./deps directory when called with `make` then build swapview binary

lilydjwg commented on broken/Chicken/Makefile:
```diff
@@ -1,8 +1,19 @@
 
-swapview: swapview.scm
-	chicken-csc -O5 swapview.scm
+CHECKEN_INSTALL_REPOSITORY:=$(shell pwd)/deps
```
CH**I**CKEN. Two lower in the exports too.

farseerfc replied:
oh, sorry. Fixed

lilydjwg commented on broken/Chicken/Makefile:
```diff
@@ -1,8 +1,19 @@
+install:
+	mkdir -p $(CHICKEN_INSTALL_REPOSITORY)
+	chicken-install format
+	chicken-install srfi-1
```
We need a timestamp file to avoid installing everytime `make` is run.

farseerfc commented:
> We need to drop the `swapview` binary in the commit.

Added a .gitignore to ignore binary

> It doesn't run without `CHICKEN_REPOSITORY_PATH`, which is annoying. Can we make it work without this?

seems passing `-static` to chicken-csc will result a binary that linked to libc but no chicken libraries.

> Also there is an error:
> 
> ```
> FORMAT: error with call: (format #f "~,1f<===~a" ===>143/128 MiB )
>         illegal character in number->string
> ```

Fixed by treating `size` in `filesize` as float instead of fixnum

lilydjwg commented:
原来那个 binary 是我加进去的啊。

我 rebase 了然而 github 好像懵了 -_-
